extensions/wikia/AdEngine/js/template/bfaaMobile.js

```diff
@@ -32,6 +33,11 @@ define('ext.wikia.adEngine.template.bfaaMobile', [
 		wrapper;
 
 	function runOnReady(iframe, params) {
+		function onVideoEndedCallback() {
+			aspectRatio = params.aspectRatio;
+			onResize();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the declaration of "onResize" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)